modules/preview-react/segmented-control/index.ts

```diff
@@ -1,2 +1,4 @@
 export * from './lib/SegmentedControl';
 export * from './lib/hooks/useSegmentedControlModel';
+export * from './lib/hooks/useSegmentedControlItem';
```

Comment:
I'm OK with fixing this but let's make sure we get this in the commit message body, I think it will show up in Release Notes then.